connectors/spark/src/main/scala/io/unitycatalog/spark/token/FixedUCTokenProvider.java

```diff
@@ -0,0 +1,26 @@
+package io.unitycatalog.spark.token;
```

Comment:
arent all of these related to auth? yeah, its not storage authentication, but UC authentication, but still its related auth right?
if so, should we move it to `auth` dir? 
`token`is a very generic term and by itself confusing and ambiguous. "auth token" makes sense. but at that point, might as well put it under `auth` package. 

if you want to be really structured, then all storage auth can go into `auth/storage` and this UC auth stuff and go into `auth/rest` or `auth/catalog`. but i am okay to keep things simple for now.